awsshell/wizard.py

```diff
@@ -71,6 +71,10 @@ def __init__(self, session=None, interaction_loader=None,
         self._interaction_loader = interaction_loader
         if interaction_loader is None:
             self._interaction_loader = InteractionLoader()
+        if delegation_loader is None:
```

Comment:
Interesting. Usually it is not the best practice to be adding an extra parameter solely for unit testing. Can you see the parameter ever being used in the future? If not, is there any other way of testing this like could just use the ``Stage`` interface directly since the wizard loader gets injected there?